### DISCUSSION TITLE
Input data type for the ImperceptibleASR attack

### DISCUSSION THREAD

**Discussion by MiguelCosta94** [Trusted-AI/adversarial-robustness-toolbox#1857] [Q&A]
Hi,

I am trying to apply the ImperceptibleASR attack on a neural network that takes as input the MFCC features of a WAV file and returns a probability distribution of the possible words. Is it possible to apply this attack on this neural network or does the attack implemented in your framework only accept estimators whose input features are the raw NumPy arrays extracted from WAV files?

Best regards,
Miguel Costa

**Comment by beat-buesser** [comment]
Hi @MiguelCosta94 Thank you very much for using ART! The current tools in ART for audio are expecting the raw Numpy arrays extracted from WAV files. Do you have a specific application or reason to use MFCCs as input?
